src/App.jsx

```diff
@@ -298,10 +359,14 @@ function KanbanBoard() {
                                   onClick={() => {
                                     const taskContent = prompt('Enter task content:');
                                     if (taskContent?.trim()) {
+                                      const dueDate = new Date();
+                                      dueDate.setDate(dueDate.getDate() + 7); // Default to 1 week from now
                                       addTask(column.id, {
                                         id: crypto.randomUUID(),
                                         content: taskContent,
-                                        createdAt: new Date(),
+                                        createdAt: dueDate.toISOString(),
+                                        dueDate: dueDate.toISOString(),
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
createdAt: format(dueDate, 'ISO_8601'),
dueDate: format(dueDate, 'ISO_8601'),
```
Date handling should use a consistent approach. Consider using a date utility library like date-fns or Moment.js for consistent date manipulation across the application.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;